app/workers/tags/alias_retag_worker.rb

```diff
@@ -0,0 +1,18 @@
+module Tags
+  class AliasRetagWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :low_priority, retry: 5
+
+    def perform(tag_id)
+      tag = Tag.find_by(id: tag_id)
+      return unless tag
+
+      tag.taggings.each do |tagging|
```

Comment:
I think it would be a good idea to use find_each here in case there are a lot of taggings

Reply:
Good catch, thanks! I'm normally more mindful of this, but it slipped through... 😃